engine/function/src/templates/Basic.ftl

```diff
@@ -842,6 +842,33 @@ public class Basic {
         return ifelseObj(new BooleanVectorDirect(condition), trueCase, falseCase);
     }
 
+    /**
+     * Copies the specified array, replacing elements that represent null in the Deephaven convention by the most
+     * recently encountered non-null value if one exists. Otherwise (if no such value exists), replaces those elements
+     * with null.
+     *
+     * @param values values.
+     * @return A copy of the specified array, with Deephaven null elements replaced as described above. If the specified
+     *         array is null, returns null.
+     */
+    public static <T> T[] forwardFillObj(final T[] values) {
```

Comment:
```suggestion
    @SafeVarargs
    public static <T> T[] forwardFillObj(final T... values) {
```
This change lets users provide args by hand.  I don't expect them to use this particular function this way, but users are creative -- and the change would be consistent with other methods.